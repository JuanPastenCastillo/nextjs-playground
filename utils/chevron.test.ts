import { expect, test } from "vitest"

test("toThrowErrorMatchingSnapshot", () => {
  expect(() => {
    throw new Error("error")
  }).toThrowErrorMatchingSnapshot("hint")
})

test("snapshot", () => {
  expect(new Error("error")).toMatchInlineSnapshot(`[Error: error]`)

  expect(() => {
    throw new Error("error")
  }).toThrowErrorMatchingInlineSnapshot(`[Error: error]`)
})
