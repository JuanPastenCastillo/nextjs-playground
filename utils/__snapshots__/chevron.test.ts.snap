// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`toThrowErrorMatchingSnapshot > hint 1`] = `[Error: error]`;
